lib/hex_web/store/local.ex

```diff
@@ -31,10 +31,19 @@ defmodule HexWeb.Store.Local do
     File.write!(Path.join(dir, "registry.ets.gz"), :zlib.gzip(data))
   end
 
+  def put_registry_signature(signature) do
+    File.mkdir_p!(dir)
+    File.write!(Path.join(dir, "registry.ets.signed"), signature)
```

Comment:
But we don't provide the ets file so it may be confusing otherwise. Also, `send_registry_signature` uses `"registry.ets.gz.signed"` so it needs to be consistent. If we had tests they would break.